### ISSUE TITLE
Add server-side routes to get MATS statistics and variables

### ISSUE THREAD

**Issue by mollybsmith-noaa** [NOAA-GSL/MATS#759]
The scorecard app will need to know what statistics and variables are available in other MATS apps, in order to display them in its selectors. Currently, this information is not available in metadata, so the apps themselves will need to provide the information via server-side routes. I propose:

<url_for_app>/getVariables
<url_for_app/getStats

Both of these routes, when entered into a browser or called by a function, should return the desired information.

**Comment by mollybsmith-noaa** [comment]
This work has been completed, and these routes will work in all apps once my pull request is merged into development.
